### ISSUE TITLE
build.cmd throws this error "LINK : fatal error LNK1181: cannot open input file 'rpc.lib'"

### ISSUE THREAD

**Issue by 1080772AntonioDidier** [microsoft/AirSim#4655]
i want to create a custom environment for simulation.

im following the guide https://microsoft.github.io/AirSim/build_windows/ to build the solution

when running the build.cmd it starts going and then throws this error.

![image](https://user-images.githubusercontent.com/3842298/182783562-1363b02c-1389-48a2-a8b4-d4d3a9d381b7.png)

the problem is within the AirLib but i can't figure this out.

am i missing some dependencies?

**Comment by dzywater** [comment]
Yes, cannot find rpc.lib. Maybe your network is, I just guess, poor.
You can open AirSim.sln and configure projects. 
rpclib: https://github.com/rpclib/rpclib/releases
